### DISCUSSION TITLE
Skin switching in game play

### DISCUSSION THREAD

**Discussion by RaoraRare** [ppy/osu#23743] [Q&A] [answered]
I just saw the new featured Artist Andora preview video and was wondering if the skin switching during the video was a feature in lazer and if not would it ever get add to the game.

**Comment by monochrome22** [comment, marked as answer]
You can select a random skin by pressing Ctrl+Shift+R during gameplay.
It seems that in the video, they rebound the random skin hotkey to Z, so when they tap X and Z, the skin is constantly changing randomly.

**Reply by RaoraRare** [reply]
I will give that a try

**Reply by peppy** [reply]
You can also hit ctrl-o to open up the settings and change it there
